e2e/node/e2e.test.ts

```diff
@@ -1014,52 +1016,65 @@ describe(`End-to-end access grant tests for environment [${environment}}]`, () =
       });
     });
 
-    it("can access a contained resource with a recursive Access Grant", async () => {
-      accessGrant = await approveAccessRequest(
-        accessRequest,
-        // Access is granted to the target container and all contained resources.
-        { inherit: true },
-        {
-          fetch: resourceOwnerSession.fetch,
-          accessEndpoint: vcProvider,
-        }
-      );
-      const requestorFile = await getFile(testFileIri, accessGrant, {
-        fetch: requestorSession.fetch,
-      });
-
-      await expect(requestorFile.text()).resolves.toBe(testFileContent);
+    const testIf = (condition: boolean) => (condition ? it : it.skip);
+    // Conditional tests confuse eslint.
+    /* eslint-disable jest/no-standalone-expect */
+
+    // Only enable this test in environments supporting recursive access grants
+    testIf(environmentFeatures?.RECURSIVE_ACCESS_GRANTS === "true")(
```

Comment:
What if we add it to our testing framework? `const featureTest = createFeatureTests("RECURSIVE_ACCESS_GRANTS", jest)`

Reply:
That's a very good idea